.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+Closes issue #number
```

Comment:
```suggestion
Closes #number

<!--
IMPORTANT:
If your PR doesn't close a particular issue, please, add the context block instead:

### Context
What you're adding/changing and why you're doing so...
-->
```

Reply:
> If your PR doesn't close a particular issue, please, add the context block instead:

yeah, this is what we are trying to avoid in general: PRs without corresponding issues

Reply:
Agreed, changed the hidden note, so that it asks to create issues with context first, and then submit a PR.